utopia-remix/app/components/next.tsx

```diff
@@ -0,0 +1,349 @@
+      overflow: 'hidden',
+    }}
+  >
+    <div
+      style={{
+        display: 'flex',
+        alignItems: 'center',
+        borderBottom: '1px solid #383C4A',
+        paddingLeft: 8,
+        paddingRight: 8,
+        height: 32,
+        minHeight: 32,
+      }}
+    >
+      <div
+        style={{
+          display: 'flex',
+          alignItems: 'center',
+          flexGrow: 1,
+          gap: 6,
+        }}
+      >
+        <EyeButton />
+        <EyeButton />
+      </div>
+
+      <div style={{ cursor: 'grab' }}>{props.title}</div>
+    </div>
+
+    <div style={{}}>{props.children}</div>
+  </div>
+)
+
+const contactUsNavigation = [
+  { name: 'Play with Utopia', href: 'https://utopia.app/p/36ae27be-welcome-to-utopia' },
+  { name: 'Join our Discord', href: 'https://discord.gg/NEEnPKCgzC' },
+  { name: 'Check us on Github', href: 'https://github.com/concrete-utopia/utopia' },
+  { name: 'Privacy Policy', href: '/policies#privacy-policy' },
+  { name: 'Terms and Conditions', href: '/policies' },
+]
+
+export const ContactUs = () => (
+  <>
+    <div
+      className='max-w-2xl h-52 md:h-36 mx-auto mb-6 font-body md:text-lg text-sm text-center'
+      style={{
+        position: 'relative',
+        border: '1px solid black',
+        padding: 16,
+      }}
+    >
+      <div
+        style={{
+          position: 'absolute',
+          left: 10,
+          right: -10,
+          top: 10,
+          bottom: -10,
+          border: '1px solid black',
+          background: 'white',
+          padding: 14,
+          display: 'flex',
+          flexDirection: 'column',
+        }}
+      >
+        <span>
+          Stay in touch. We’ll send you a long-form email once or twice a month <br />
+          (and only when we have enough new development to write about!)
+        </span>
+        <BasicEmailSignup />
+      </div>
+    </div>
+    <div className='font-body md:text-lg text-sm flex justify-center items-center'>
+      {contactUsNavigation.map((item, index) => (
+        <a
+          key={`contact-us-nav-${index}`}
+          href={item.href}
+          className='block px-6 py-2 rounded-md text-body hover:text-gray-900 hover:bg-gray-50 text-center'
+          onMouseDown={() =>
+            gtag('event', 'navigate', { category: 'links', label: item.href, value: 1 })
+          }
+        >
+          {item.name}
+        </a>
+      ))}
+    </div>
+  </>
+)
+
+declare function gtag(...args: any): void
+
+function enableCookies() {
+  gtag('consent', 'update', {
+    analytics_storage: 'granted',
+  })
+}
+
+export const CookieConsentBar = () => {
+  if (getCookieConsentValue() === 'true') {
+    enableCookies()
+  }
+
+  return (
+    <CookieConsent
+      location='bottom'
+      style={{ background: '#383C4A', fontSize: '13px' }}
+      buttonStyle={{
+        color: '#383C4A',
+        fontSize: '13px',
+        borderRadius: 5,
+        backgroundColor: '#00FC00',
+      }}
+      //   onAccept={enableCookies}
```

Comment:
Is this meant to still be commented out?

Reply:
pfft, good catch it was a remnant of the component fixes. done in [`9837e3e` (#5008)](https://github.com/concrete-utopia/utopia/pull/5008/commits/9837e3ef8b6115a6faa907b4d0596664d77f9b4f)